pkg/props/cell.go

```diff
@@ -1,11 +1,11 @@
-// Package contains all props used to customize components.
 package props
 
 import (
 	"github.com/johnfercher/maroto/v2/pkg/consts/border"
 	"github.com/johnfercher/maroto/v2/pkg/consts/linestyle"
 )
 
+// Cell is the representation of a cell in a table.
```

Comment:
// Cell is the representation of a cell in a the grid system.

Reply:
done